components/intake-form/WorkExperience.tsx

```diff
@@ -210,6 +216,9 @@ export default function WorkExperience({
             </div>
           </SelectableGrid>
         </div>
+        {showSuggestButton ? null : (
+          <div style={{ margin: "1rem", color: "#DC143C" }}>{labelNote}</div>
```

Comment:
Would you be open to using the `<ErrorMessage />` component? It comes with some styling that feels pretty close to what youʻre trying to achieve here.

Hereʻs what it could look like instead:
<img width="742" alt="image" src="https://user-images.githubusercontent.com/5141111/187861334-76948881-d1a3-41f6-b2f5-50068c87b651.png">

Reply:
makes sense, will add that in the next commit

Reply:
Done with the latest commit